.bazelci/presubmit.yml

```diff
@@ -28,6 +28,8 @@ buildifier:
   build_targets:
     - "--"
     - "..."
+    # This has its own workspace and needs to run from there
+    - "-//examples/py_proto_library/..."
```

Comment:
The targets in `//examples/py_proto_library/...` are still matched by `...`, failing the CI because the proto_library tries to strip `example.com`, but the path starts with `examples/py_proto_library`

Reply:
I would expect that changes proposed in https://github.com/linzhp/rules_python/pull/1 would remove the need for this change.